docs/user/template-examples.md

```diff
@@ -2,6 +2,17 @@
 description: Get inspired with a variety of UI template examples in Node-RED Dashboard 2.0, enhancing your dashboard design process.
 ---
 
+
+<script setup>
```

Comment:
I'll merge, but there might be a nice trick with putting the mapping from worldmap to ExampleFlowWorldmap in the frontmatter instead?

Reply:
This is frontmatter essentially, but allows me to write JS, instead of formatted yml